### DISCUSSION TITLE
who to set up WebDriver path?

### DISCUSSION THREAD

**Discussion by edenhanan** [seleniumbase/SeleniumBase#2714] [Q&A] [answered]
hello, can I set up a specific driver path like in the original Selenium? 
example: 
`driver = webdriver.Chrome('/path/to/chromedriver')`
I can't use the default way or `seleniumbase get chromedriver`  command, because I need to run the test on a computer without internet access, I can transfer files in and out of the computer but without a connection I always get an error to update my webdriver.
it doesn't have to use the driver object like in the original Selenium, I just want to point to a specific driver path for seleniumbase to use.

**Comment by mdmintz** [comment, marked as answer]
The driver either needs to be on your System PATH, or in the `seleniumbase/drivers` folder for where SeleniumBase is installed. (For UC Mode, it can only be in the `seleniumbase/drivers` folder.)

Mismatched driver/browser pairings are allowed if you pass `--driver-version="keep"` as a `pytest` arg, or `driver_version="keep"` in the `SB()` / `Driver()` formats.
